lavalink/websocket.py

```diff
@@ -43,6 +47,9 @@ def connected(self):
             'User-Id': str(self._user_id)
         }
 
+        if self._resuming_configured and self._resume_key is not None:
```

Comment:
Implicit here ?
Can `_resume_key` be an empty string ? :thonk:

Reply:
I'd rather keep it as `None` which is the equivalent to Java's `null`, and Lavalink interprets `null` as disabling the resume functionality.

Reply:
I meant, if `_resume_key` isn't supposed to be an empty string, maybe you can use 
`and self._resume_key`
instead of 
`and self._resume_key is not None`

Reply:
Ah yeah